README.rst

```diff
@@ -205,16 +205,23 @@ If you'd like to help with the project itself, here are some other ways you can
 
 Testing
 ~~~~~~~
+This role includes a Vagrantfile used with a Docker-based test harness for integration testing. Using Vagrant allows all contributors to test on the same platform and avoid false test failures due to untested or incompatible docker versions.
```

Comment:
This role used to use Vagrant for testing, but I switched to Docker because it uses much fewer resources.

Using Vagrant would indeed help prevent a specific test failure mode, as you've pointed out in the README.

Could you rephrase this to imply using the Vagrant box is optional? Maybe something like "We also offer a Vagrant box that approximates the Travis CI setup. Here's how to use it:".